browser/src/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { App } from './App';
+import './index.css';
```

Comment:
I don't love the idea of coupling our CSS so closely with our React -- can you move CSS to our HTML and use SCSS instead?

Reply:
Sure. Can you elaborate on why? I'm not sure why this might be an issue as-is, and what changing it to your suggestion fixes.

Reply:
The benefit of CSS in JS is CSS will load with your components, but this is an index.css file being pulled into an index.js file so it'll load no matter what if your JS is loaded.

The issue is that you're conflating your assets.  Keeping things as separate as possible is generally a good rule of thumb and in this case you're combining your CSS and JS strategies.  Beyond that, this will now require a bundler to work, because CSS can't natively be imported into JS.  Ideally applications can work bundler-free.  You'll need them in some cases, especially if you're using something like JSX that requires transpilation, but beyond that the ideal is that you could run your app as-is and it'd work in the browser.

Reply:
Gotcha! Thanks for explaining that. That makes sense. I'll rework this soon